CONTRIBUTING.md

```diff
@@ -0,0 +1,116 @@
+# FreeCodeCamp forum clone's contributing guide
+
+## Documentation
+
+Please, make sure to read:
+
+- our [code of conduct](./CODE_OF_CONDUCT.md)
+- and [README](./README.md)
+
+## Issues
+
+### Creating issues
+
+Feel free to check out [our codebase](https://github.com/jdwilkin4/fcc-forum-clone). If you think, that something could be added, fixed, improved - make sure, that there is no open issue for that already by going [here](https://github.com/jdwilkin4/fcc-forum-clone/issues).
+
+If you are confident, that your issue won't be a duplicate - go ahead and [create it](https://github.com/jdwilkin4/fcc-forum-clone/issues/new/)!
+
+If you have some ideas about possible changes, but not sure enough, please share your thoughts by creating new thread in [the discussions](https://github.com/jdwilkin4/fcc-forum-clone/discussions).
+
+### Picking up an issue to work on
+
+Browse through [opened issues](https://github.com/jdwilkin4/fcc-forum-clone/issues). Feel free to start working on any issue which does not have have assignee yet!
+
+If you are [collaborator](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-personal-account-settings/permission-levels-for-a-personal-account-repository#collaborator-access-for-a-repository-owned-by-a-personal-account) - assign yourself to the issue you want to work on.
+
+If you are not collaborator - would be better to left a comment in the issue thread, so we know that you are working on it.
+
+In any case - make sure to use the issue thread to ask questions about the issue if you need!
+
+## Making changes
+
+If you are experienced with Git workflow, **How to contribute** section of our [README](./README.md) may be enough to work on the issue you picked up!
+
+If you are somewhat new to Git and GitHub - in the below will be couple more tips and useful links about all this!
+
+### Installing git
+
+If you don't have git on your machine, [install it](https://help.github.com/articles/set-up-git/).
+
+### Installing dependencies
+
+Instructions about this can be found in [README](./README.md).
```

Comment:
This section should be moved further down. It should be after the section about cloning the project. 
Also let's give them some instructions on how to install Node.js in order to use the `npm` command. 

Proposed fix: 
If you don't have Node.js installed on your computer, please read through [these instructions](https://nodejs.dev/en/).  If you are not sure if Node.js is installed, please run the `node -v` command in the command line. 

Then change to the project directory by using the `cd fcc-forum-clone` command. To install the dependencies, run the `npm install` command.